Gwayeon/Controllers/HomeViewController.swift

```diff
@@ -77,15 +76,25 @@ class HomeViewController: UIViewController {
         super.viewDidLoad()
         
         configureViewComponent()
+        
+        Task { [weak self] in
+            self?.recommendFruits = await FirebaseManager.shared.getFruits()
+            print(self?.recommendFruits)
+            
+            DispatchQueue.main.async {
+                self?.collectionViewFlowLayout.collectionView?.reloadData()
```

Comment:
```suggestion
                self?.listCollectionView.reloadData()
```
으로 수정하는 건 어떨까요? reload시에, FlowLayout에서 collectionView가 reload되는걸 사용할 수 있다는 걸 처음 배웠습니다.. !